docs/fastdds/api_reference/spelling_wordlist.txt

```diff
@@ -85,3 +85,34 @@ statusmask
 PublicationMatchedStatus
 statusmask
 SubscriptionMatchedStatus
+Unregister
+Otherwhise
```

Comment:
Not a valid word

Reply:
As a reply to all the comments:
1. I think you're not looking at the last commit, I believe I've taken care of all duplicates
2. Actually a lot of wrong words made the list. The problem is that they are wrong in the source code doxygen documentation of the header files themselves. I'd like to have them corrected in Fast itself, but until then I've added them to the dictionary so we don't ge hundreds of warnings we are not causing.